cmake/FindCUDA/FindCUDA.cmake

```diff
@@ -686,21 +738,17 @@ endif()
 
 
 # CUDA_NVCC_EXECUTABLE
-if(DEFINED ENV{CUDA_NVCC_EXECUTABLE})
-  SET(CUDA_NVCC_EXECUTABLE "$ENV{CUDA_NVCC_EXECUTABLE}")
```

Comment:
this change shouldn't be deleted. our ccache fork depends on this.

Reply:
Upstream PR: https://gitlab.kitware.com/cmake/cmake/merge_requests/1899